packages/web/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+export {}
```

Comment:
What does the `export {}` do here?

Reply:
If you don't export, typescript doesn't like declaring the global type (because it uses namespace). AFAIK,  export tells typescript that its a module type declaration and allows the use of declare namespace

Reply:
Without the export 
![Uploading Screenshot 2021-02-02 at 17.58.43.png…]()